lib/engine/player.rb

```diff
@@ -15,8 +15,9 @@ class Player
     attr_accessor :bankrupt
     attr_reader :name, :companies
 
-    def initialize(name, count_companies: true)
+    def initialize(name, game, count_companies: true)
```

Comment:
we should avoid this, models shouldn't take games

can you instead put a property on the share?

Reply:
Sure. Have pushed a refactored version.